templates/docs/base/code.md

```diff
@@ -42,37 +42,43 @@ The code snippet pattern should be used to group related code examples, making t
 
 **Basic code snippet**
 
-<div class="embedded-example"><a href="/docs/examples/patterns/code-snippet/code-snippet" class="js-example">
+<div class="embedded-example"><a href="/docs/examples/patterns/code-snippet/default" class="js-example">
 View example of the code snippet
 </a></div>
 
 **Code snippet with icon**
 
 A code snippet block can include an icon by using the `.p-code-snippet__block--icon` class. By default, it will show the Linux prompt icon, but other icon options are available, using the `.is-windows-prompt` or `.is-url` classes.
 
-<div class="embedded-example"><a href="/docs/examples/patterns/code-snippet/code-snippet-icon" class="js-example">
+<div class="embedded-example"><a href="/docs/examples/patterns/code-snippet/icon" class="js-example">
 View example of the code snippet
 </a></div>
 
 **Code snippet with numbered lines**
 
-<div class="embedded-example"><a href="/docs/examples/patterns/code-snippet/code-snippet-numbered" class="js-example">
+<div class="embedded-example"><a href="/docs/examples/patterns/code-snippet/numbered" class="js-example">
 View example of the code snippet
 </a></div>
 
 **Code snippet with dropdown**
```

Comment:
We have quite a few of these subtitles. Maybe it would make sense to actually make them proper headings with `#### Code snippet ...`?

Reply:
@bartaz should we move it out of the "Base" category, to "Components"? I agree with you and I'm changing these to headings, but maybe it would be good to have these sections visible in the side nav - which wouldn't happen while "Code snippet" is a child of the "Code" section.

Reply:
While, as discussed, we should restructure the headings to show variants of code snippet in side nav, I think we should still keep in in 'Base'.

It is a bit confusing, but would be weird to separate pages for 'Code' base styles and 'Code snippet' component.
In similar way 'Form' page contains both base styling for forms and components that we use with forms. Or "Table" page is about all things tables.